checkov/main.py

```diff
@@ -287,8 +287,8 @@ def run(self, banner: str = checkov_banner, tool: str = checkov_tool, source_typ
                 repo_root_for_plan_enrichment=self.config.repo_root_for_plan_enrichment,
                 resource_attr_to_omit=self.config.mask,
                 # TODO modify the output for git_history secret and remove the rewrite of enable_git_history_secret_scan
-                # enable_git_history_secret_scan=self.config.scan_secrets_history,
-                enable_git_history_secret_scan=False,  # expose after unite git history with secret scan
+                enable_git_history_secret_scan=self.config.scan_secrets_history,
+                # enable_git_history_secret_scan=False,  # expose after unite git history with secret scan
```

Comment:
Do we want to open it? I think we need to open it only after Maryna's test

Reply:
damn.. pushed by accident again